cassdegrees/api/models.py

```diff
@@ -11,16 +11,34 @@ class SampleModel(models.Model):
 class CourseModel(models.Model):
     id = models.AutoField(primary_key=True)
     code = models.CharField(max_length=32)
-    year = models.PositiveIntegerField()
     name = models.CharField(max_length=256)
     units = models.PositiveIntegerField()
+
+    offered_years_choices = [
+        ("ALL", "Every Year"),
+        ("ODD", "Odd Years"),
+        ("EVEN", "Even Years"),
+        ("OTHER", "Other/Unknown")
+    ]
+
+    offeredYears = models.CharField(choices=offered_years_choices, max_length=5)
+
     offeredSem1 = models.BooleanField()
     offeredSem2 = models.BooleanField()
+    offeredSummer = models.BooleanField()
+    offeredAutumn = models.BooleanField()
+    offeredWinter = models.BooleanField()
+    offeredSpring = models.BooleanField()
+
+    otherOffering = models.BooleanField()
+
+    currentlyActive = models.BooleanField()
+
     rules = psql.JSONField(default=list)
     lastUpdated = models.DateField(default=timezone.now)
 
     class Meta:
-        unique_together = (("code", "year"),)
+        unique_together = (("code", "name"),)
```

Comment:
Not sure about this one - is there any value for name also being unique?

Reply:
According to Jack, CASS recycles course codes so there will be different courses with same course code. 
Oh and also, this belongs in the other pull request:
https://github.com/cass-degrees/CASS-Degrees-Code/pull/309